src/app/Scenes/HomeView/Components/RecommendedAuctionLotsRail.tsx

```diff
@@ -48,15 +47,14 @@ export const RecommendedAuctionLotsRail: React.FC<RecommendedAuctionLotsRailProp
 
     return (
       <View ref={railRef}>
-        <Flex pl={2} pr={2}>
-          <SectionTitle
-            title={title}
-            onPress={() => {
-              trackEvent(tracks.tappedHeader(contextScreenOwnerType))
-              navigate("/auctions/lots-for-you-ending-soon")
-            }}
-          />
-        </Flex>
+        <SectionTitle
+          mx={2}
+          title={title}
+          onPress={() => {
+            trackEvent(tracks.tappedHeader(contextScreenOwnerType))
+            navigate("/auctions/lots-for-you-ending-soon")
+          }}
```

Comment:
To enable prefetching 🚀 

```suggestion
          href="/auctions/lots-for-you-ending-soon"
          onPress={() => {
            trackEvent(tracks.tappedHeader(contextScreenOwnerType))
          }}
```